### PR TITLE
Minor Website UI Updates

### PR DESCRIPTION
**Pull request by n3tn1nja**
Visual Changes: 
- Display the value of a user's locked SNOB. 
- Display 4 decimal points if SNOB is worth less than 50 cents. 
- Change the Vote Allocation Pie chart to a Bar chart for a better page layout. 

Technical Changes:
- Refactor `formatNumber()` default `precision` argument.

### CODE REVIEW

File: Layout/Shared/SnobPrice/index.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const SnobBalance = () => {
             color='textPrimary'
             className={classes.balance}
           >
-            $ {formatNumber(prices.SNOB, 2)}
+            $ {prices.SNOB > 0.5 ? formatNumber(prices.SNOB) : formatNumber(prices.SNOB, 4)}
```
Comment (jomarip) [inline Layout/Shared/SnobPrice/index.js:52]: The increasing of decimal places for the Snowball prices seems fine.

File: Layout/Shared/ThemeButton/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Switch } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
 import { useDarkMode } from 'contexts/ui-context';
-import { AnalyticCategories, createEvent, analytics } from "utils/analytics";
+import { AnalyticCategories, createEvent, useAnalytics } from "contexts/analytics";
```
Comment (jomarip) [inline Layout/Shared/ThemeButton/index.js]: Working on following the impace of the change from analytics to UseAnalytics with a new folder as reference.

File: components/WalletModal/index.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ import WalletCard from 'components/WalletModal/WalletCard'
 import { walletlink, injected } from 'utils/constants/connectors'
 import ANIMATIONS from 'utils/constants/animate-icons'
 
-import { AnalyticActions, AnalyticCategories, createEvent, analytics } from "utils/analytics";
+import { AnalyticActions, AnalyticCategories, createEvent, useAnalytics } from "contexts/analytics";
```
Comment (jomarip) [inline components/WalletModal/index.js]: Not sure if this is the cause, but I was getting an error when connecting initially and the preview is not acting like it is not on Avalanche NEtwork. Could be something with the preview generation though. I have to dig into it.

image

File: contexts/analytics.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
+import { MatomoProvider, createInstance, useMatomo } from '@datapunt/matomo-tracker-react'
```
Comment (jomarip) [inline contexts/analytics.js]: @Jonasslv or @saito-sv you have anything thoughts about this new analytics page created?

**Review by jomarip — COMMENTED** [review]
Looks good to me. Let me have 1 other person look at it, but yeah, this should be good to roll unless I missed anything.

**Comment by ghost** [comment]
You will need to rebase those changes on top of dev, we shouldn't have all this merge commits on dev.

**Comment by n3tn1nja** [comment]
@Jonasslv @jomarip I rebased the branch onto development.  Let me know if everything is good.

**Review by jomarip — APPROVED** [review]
Allowing the merge into development to keep things moving. We should be able to test from the preview more.

**Pull request merged**